modules/serverless-functions-func-yaml-environment-variables.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-functions-func-yaml-environment-variables_{context}"]
+= Referencing environment variables from func.yaml fields
+
+In any of the fields in `func.yaml`, you can put a reference to an environment variable available in the local environment. This can be useful for avoiding storing sensitive information such as an API key in the function configuration.
```

Comment:
Maybe better confirm with  @zroubalik if this can be applied to any field. I tried that with other fields as `namespace` or on annotations, and it did not work.

Reply:
@zroubalik Do you think we should change the text from:
  In any of the fields in `func.yaml`, you can put a reference
to
  Anywhere in the `envs` field of `func.yaml`, you can put a reference